app/Models/Contact/Contact.php

```diff
@@ -223,7 +223,7 @@ public function tasks()
     /**
      * Get the tags records associated with the contact.
      *
-     * @return HasMany
+     * @return BelongsToMany
```

Comment:
Why? It really is a HasMany relationship.

Reply:
see above